Money*/Money*/Utils/Constant.swift

```diff
@@ -15,6 +15,14 @@ struct Constant {
     static let shadowRadius = CGFloat(2)
     static let shadowOffset = CGSize(width: 0, height: 1)
     static let shadowOpacity = Float(0.6)
+    static let walletCollectionMinimumLineSpacing = CGFloat(40)
```

Comment:
```suggestion
    static let walletCollectionMinimumLineSpacing: CGFloat = 40 
```
nên khai báo như thế này anh nhé, anh sửa lại các biến luôn, dùng kiểu kia chỉ cho trường hợp ép kiểu thôi